### PR TITLE
Add Ballerina class support and fix API doc comment indentation

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
$subject

## Approach
- Add class support
- Indent API doc comments

## Related PRs
https://github.com/ballerina-platform/ballerina-lang/pull/25637

## Test environment
- SLP4
- Java 8
- macOS

### CODE REVIEW

File: io-native/src/test/java/org/ballerinalang/stdlib/io/characters/AsyncReadWriteTest.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public void setup() {
         currentDirectoryPath = System.getProperty("user.dir") + "/build/";
     }
 
-    @Test(description = "Tests reading characters through the async io framework")
+    @Test(enabled = false, description = "Tests reading characters through the async io framework")
```
Comment (ThisaruGuruge) [inline io-native/src/test/java/org/ballerinalang/stdlib/io/characters/AsyncReadWriteTest.java:47]: Why do we disable all the tests?

Comment (BuddhiWathsala) [reply io-native/src/test/java/org/ballerinalang/stdlib/io/characters/AsyncReadWriteTest.java:47]: We have to fix this Windows build https://github.com/ballerina-platform/module-ballerina-io/issues/4

Comment (niveathika) [reply io-native/src/test/java/org/ballerinalang/stdlib/io/characters/AsyncReadWriteTest.java:47]: We can only disable in windows build right? ./gradlew clean build -x test

Comment (ldclakmal) [reply io-native/src/test/java/org/ballerinalang/stdlib/io/characters/AsyncReadWriteTest.java:47]: IMO also, better to disable in windows build only for now. Running these tests even in a single environment would be better than skipping in both.

Comment (BuddhiWathsala) [reply io-native/src/test/java/org/ballerinalang/stdlib/io/characters/AsyncReadWriteTest.java]: Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/5/commits/3e367134f7fd716ba32095be6a93ee260149df62

**Review by ldclakmal — APPROVED** [review]

**Pull request merged**